### PR TITLE
Final Changes And Verifying Code

### PR DESCRIPTION
**Pull request by sansax27**

### CODE REVIEW

File: FusionIIIT/applications/iwdModuleV2/migrations/0001_initial.py [hunk 1]
```diff
@@ -144,58 +65,154 @@ class Migration(migrations.Migration):
...
+        ),
+        migrations.CreateModel(
+            name='Addendum',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('issueDate', models.DateField()),
+                ('nitNiqNo', models.IntegerField()),
+                ('name', models.CharField(max_length=200)),
+                ('openDate', models.DateField()),
+                ('openTime', models.TimeField()),
```
Comment (saturn-vipul) [inline FusionIIIT/applications/iwdModuleV2/migrations/0001_initial.py]: Remove migrations from the PR

Comment (sansax27) [reply FusionIIIT/applications/iwdModuleV2/migrations/0001_initial.py]: Done

Comment (sansax27) [reply FusionIIIT/applications/iwdModuleV2/migrations/0001_initial.py]: Please do check that deleting init.py also does not create any problem.

**Review by Naman2018157 — APPROVED** [review]
Reveiwed

**Review by sansax27 — COMMENTED** [review]
Everywhere in case of conflicts, accept incoming merge changes, and not of head.

**Pull request merged**